packages/directives/vue-visible-on-scroll/src/index.js

```diff
@@ -0,0 +1,91 @@
+let element;
+let routeWithHash;
+let desktopBreakpoint;
+
+export default {
+  // Vue 3
+  mounted(el, binding) {
+    initOnMounted(el, binding);
+  },
+
+  // Vue 2
+  inserted(el, binding) {
+    initOnMounted(el, binding);
+  },
+
+  // Vue 3
+  unmounted() {
```

Comment:
What about Vue 2? If `unbind` is the equivalent, let's add that here for completeness.